packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -260,42 +283,21 @@ export class UnreferencedStateTracker {
         // Also, start a timer for the sweep timeout.
         if (unreferencedDurationMs >= this.inactiveTimeoutMs) {
             this._state = UnreferencedState.Inactive;
-            this.clearTimers();
```

Comment:
This may be the only actual behavior change, now that I think about it.  `this.sweepTimer` used to get cleared then `start`ed every time when we were past `inactiveTimeoutMs`, but with this change it'll go through `restart` instead.